toolkit/imageconfigs/cvm.json

```diff
@@ -0,0 +1,73 @@
+{
+    "Disks": [
+        {
+            "PartitionTableType": "gpt",
+            "MaxSize": 2048,
+            "Artifacts": [
+                {
+                    "Name": "cvm",
+                    "Type": "vhd"
+                }
+            ],
+            "Partitions": [
+                {
+                    "ID": "efi",
+                    "Flags": [
+                        "esp",
+                        "boot"
+                    ],
+                    "Start": 1,
+                    "End": 513,
+                    "FsType": "fat32",
+                    "TypeUUID": "c12a7328-f81f-11d2-ba4b-00a0c93ec93b"
```

Comment:
It might be goodness to also provide an alternative API that provides friendly names for the GUIDs:

e.g.

```json
"Type": "esp"
```

or:

```json
"Type": "linux-root-amd64"
```

Reply:
Agreed, but I don't think we can set that via `parted` at the moment: https://www.gnu.org/software/parted/manual/parted.html#type

Reply:
hmm maybe we can maintain our own map for these values..

Reply:
Yeah, my suggestion was to maintain our own list. (But keep `TypeUUID` as a fallback.)

Reply:
Second this, bit of boilerplate but we can add a `filesystemtype.go` (name pending) similar to `partitionflag.go`

Reply:
Added a `Type` parameter, along with a limited list type name -> type UUID of common types